www/_posts/2017-09-08-ios-release.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+author:
+    name: Shazron Abdullah
+    url: https://twitter.com/shazron
+title:  "Cordova iOS 4.5.0"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce a minor version of `Cordova iOS 4.5.0` has been released!
+
+Three new features were added:
+1. [CB-12937](https://issues.apache.org/jira/browse/CB-12937) - Plugins can receive `handleOpenURLWithApplicationSourceAndAnnotation:` now (new selector, that sends the URL with additional metadata) 
+2. [CB-13164](https://issues.apache.org/jira/browse/CB-13164) - Integrated cordova-plugin-console to build in support for window.console. 
+3. [CB-10916](https://issues.apache.org/jira/browse/CB-10916) - Support [display name](/docs/en/dev/config_ref/index.html#name) for **iOS**
+
+Note that for [CB-13164](https://issues.apache.org/jira/browse/CB-13164) if you already included `cordova-plugin-console` in your project, you must **remove** that plugin if not your project will not build.
```

Comment:
Perhaps

> **Important!** If you have included `cordova-plugin-console` in your project, you must **remove** it, otherwise your project will not build.

Reply:
Agreed, this needs to be called out more prominently

Reply:
Yeah, it ends up with duplicate symbol linker errors:

```
duplicate symbol _OBJC_CLASS_$_CDVLogger in:
    /Users/me/Library/Developer/Xcode/DerivedData/Project/Build/Intermediates/ArchiveIntermediates/Project/IntermediateBuildFilesPath/Project.build/Release-iphoneos/Project.build/Objects-normal/arm64/CDVLogger.o
    platforms/ios/build/device/libCordova.a(CDVLogger.o)
duplicate symbol _OBJC_METACLASS_$_CDVLogger in:
    /Users/me/Library/Developer/Xcode/DerivedData/Project/Build/Intermediates/ArchiveIntermediates/Project/IntermediateBuildFilesPath/Project.build/Release-iphoneos/Project.build/Objects-normal/arm64/CDVLogger.o
    platforms/ios/build/device/libCordova.a(CDVLogger.o)
ld: 2 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Reply:
Thanks all -- I'll add the change to make it more prominent.

Reply:
Changes added.